### DISCUSSION TITLE
re-building after small changes

### DISCUSSION THREAD

**Discussion by choidami** [google/or-tools#2463] [Build and configuration issues] [answered]
Hello, 

I'm sorry if this is a silly question-- I'm not very familiar with build/make files.
If I make changes to one part of the C++ portion of the codebase (for example, one of the folders within ortools), is there a way to quickly re-compile without having to re-install everything, which takes a long time?

Thank you

**Comment by lperron** [comment, marked as answer]
the 3 build systems (make, cmake, and bazel) are incremental.
